tests/testthat/test-autosyn.R

```diff
@@ -82,3 +81,29 @@ test_that("fafbseg.sqlitepath is respected",{
   withr::with_options(list('fafbseg.sqlitepath'=td),
                       expect_equal(local_or_google("test.db"), tf))
 })
+
+test_that("flywire_neurons_add_synapses works", {
+  token=try(chunkedgraph_token(), silent = TRUE)
+  skip_if_not_installed('reticulate')
+  skip_if(inherits(token, "try-error"),"Skipping live flywire tests")
+  skip_if_not(reticulate::py_module_available("cloudvolume"),
+              "Skipping live flywire tests requiring python cloudvolume module")
+  expect_is(fwskel <- readRDS(testthat::test_path("testdata/flywire_neuron_skeleton.rds")), 'neuronlist')
```

Comment:
@alexanderbates please note I removed the mesh from this neuron reducing it by 10-20x in size. I try to keep test data as small as possible because it affects the installed size of the package.

Reply:
The fact I got confused here, leans me towards changing the default for skeletor as not including the mesh

Reply:
OK. I see you've done that now. I think it's probably a good change, but honestly I wouldn't have done it on this branch as it's a logically separable point.